lib/actions/field-trip.js

```diff
@@ -249,17 +406,355 @@ function planInbound (request) {
     }, config)
     // clearTrip()
     const queryParams = {
-      date: moment(request.travelDate).format(OTP_API_DATE_FORMAT),
+      date: getFormattedRequestTravelDate(request),
       departArrive: 'DEPART',
-      groupSize: getGroupSize(request),
       time: moment(request.leaveDestinationTime).format(OTP_API_TIME_FORMAT),
       ...locations
     }
     dispatch(setQueryParam(queryParams))
-    dispatch(routingQuery())
+    dispatch(makeFieldTripPlanRequests(request))
+  }
+}
+
+/**
+ * Makes appropriate OTP requests until enough itineraries have been found to
+ * accommodate the field trip group.
```

Comment:
There's some complexity to this action that I think would be helpful if included in this method's jsdoc.

Reply:
Added more comments in a4480936984a2a1cb2d9dd6f97adc4ecd3a9a12c